Gruntfile.js

```diff
@@ -56,26 +56,56 @@ module.exports = function(grunt) {
     //
     grunt.registerTask("db-populate", "Populates the database with dummy data.", function() {
         var mongoose = require("mongoose");
+        var async = require("async");
         var dbConfig = require("./config/db");
-        var Contact = require("./server/models/contact").Contact;
         
-        var dummydata = grunt.file.readJSON("./config/development-data.json").allContacts;
+        var jsonContacts = grunt.file.readJSON("./config/development-data.json").allContacts;
         var done = this.async();
         
+        mongoose.Promise = global.Promise;
+        
+        global.autoIncrement = require("mongoose-auto-increment");
         mongoose.connect(dbConfig.url);
         var connection = mongoose.connection;
+        autoIncrement.initialize(connection);
+        
+        var Contact = require("./server/models/contact").Contact;
+        
+        var contacts = [];
+        for(var i = 0; i < jsonContacts.length; i++) {
```

Comment:
This was required since I swapped to Mongoose because I need an actual Contact object.